libbeat/scripts/Makefile

```diff
@@ -261,8 +261,8 @@ testsuite: clean update
 	# Runs system and system integration tests if SYSTEM_TESTS is set to true
 	if [ $(SYSTEM_TESTS) = true ]; then \
 		if [ $(TEST_ENVIRONMENT) = true ]; then \
-        	$(MAKE) system-tests-environment; \
-    	else \
+		$(MAKE) system-tests-environment; \
```

Comment:
are changes here intentional?

Reply:
Nah. Editor trimmed it.

Reply:
Can you revert it as based on the diff I'm not 100% sure on what the change is and I want to make sure it doesn't break anything. Makefiles can be picky about indentation.